docs/guides/run-jobs-session.mdx

```diff
@@ -112,41 +131,58 @@ print(f"Result2: {job2.result()}")
 session.close()
 ```
 
-<span id="cancel"></span>
-## Cancel a session
+<span id="session-status"></span>
+## Check session status
 
-Canceling a session immediately closes it, failing all queued jobs and preventing new submission. Use the `session.cancel()` method to cancel a session.  Any jobs that are already running continue to run but queued jobs are put into a failed state and no further jobs can be submitted to the session. This is a convenient way to quickly fail all queued jobs within a session.
+You can query a session's status to understand its current state by using  `session.status()` or on the Jobs page for your channel.
+
+Session status can be one of the following:
+
+- `Pending`: The session has not started or has been deactivated. The next session job needs to wait in the queue like other jobs.
+- `In progress, accepting new jobs`: The session is active and accepting new jobs.
+- `In progress, not accepting new jobs`: The session is active but not accepting new jobs. Job submission to the session is rejected, but outstanding session jobs will run to completion. The session is automatically closed once all jobs finish.
+- `Closed`: The session's maximum timeout value has been reached or the session was explicitly closed.
+
+<span id="session-details"></span>
+## Determine session details
+
+For a comprehensive overview of a session's configuration and status, use the  `session.details() method`.
 
 ``` python
+from qiskit_ibm_runtime import QiskitRuntimeService, Session, EstimatorV2 as Estimator
+
+service = QiskitRuntimeService()
+backend = service.least_busy(operational=True, simulator=False)
+
 with Session(backend=backend) as session:
     estimator = Estimator()
-    job1 = estimator.run(...)
-    job2 = estimator.run(...)
-    # You can use session.cancel() to fail all pending jobs, for example,
-    # if you realize you made a mistake.
-    session.cancel()
+    job = estimator.run([(circuit, observable)])
+    print(session.details())
```

Comment:
It doesn't let me put in a suggestion, but same as batch - you don't need to run jobs to get session details anymore!